src/main/java/service/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation
+                    = FruitTransaction.Operation.valueOf(parts.toString());
+            String fruit = Arrays.toString(parts);
```

Comment:
Similar to the previous comment, `Arrays.toString(parts)` will give you a string representation of the array. To get the fruit name, you should use `parts[1]` if the fruit name is the second element in the CSV.